src/Command/Anonymization/AnonymizeCommand.php

```diff
@@ -237,7 +239,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     private function doBackupInitialDatabase(): void
     {
-        $this->io->section('Start backuping local database');
+        if ($this->io->isVerbose()) {
+            $this->io->section('Start backing up local database');
```

Comment:
`"` instead of `'` to keep consistent with the rest? :slightly_smiling_face:

Reply:
I made the opposit :)